VERSION

```diff
@@ -0,0 +1 @@
+0.8.2
```

Comment:
Could you change this to 0.9.0-dev? I know, this one was my fault for not changing it earlier...